app/views/provider_interface/application_choices/index.html.erb

```diff
@@ -16,7 +16,10 @@
     <tbody class='govuk-table__body'>
       <% @application_choices.each do |application_choice| %>
       <tr class='govuk-table__row'>
-        <th class='govuk-table__cell'><%= govuk_link_to application_choice.application_form.full_name, provider_interface_application_choice_path(application_choice) %></th>
+        <td class='govuk-table__cell'>
+          <span class="govuk-caption-m govuk-!-font-size-16"><%= application_choice.application_form.support_reference %></span>
```

Comment:
I guess the `govuk-!-font-size-16` (which was there before this PR in the following markup) makes the caption non-responsive?